lib/index.js

```diff
@@ -32,7 +32,11 @@ $(document).ready(() => {
     var fileName = location.pathname.split('/').slice(-1)[0]
     if (fileName === 'foods.html' && event.target.nodeName != 'BUTTON') {
       var foodId = event.target.parentElement.attributes.data.value.split('-')[1]
```

Comment:
We could probably also line 34-36 into another function that simply sets these variables, and then also pull out the if statement into another that simply checks for any change in original value upon focus out.

Reply:
I think this works for now and we can continue to pull out on later iterations after the completion of the express backend transition.